### PR TITLE
Upgraded project to PureScript v0.9.1

### PR DESCRIPTION
**Pull request by daniel-chambers**
We're looking to use purescript-webdriver in our project; but our project is using PureScript 0.9. This project is a dependency of purescript-webdriver, which I'm also looking at upgrading to PureScript 0.9 if I can. So I've first upgraded this project to PureScript 0.9 and upgraded its dependent libraries to the latest version which are compatible with 0.9.

The biggest change is in the unit tests which have been restructured to use the latest purescript-test-unit, which now has direct support for tests in `Aff`. Parallel types were also moved into purescript-parallel, so I switched the tests to use those.

I also added bower as a dev dependency, as I required it to install the dependent packages. I assume you guys must have it globally installed, but I try to avoid implicit global installation of packages where possible; hence me adding it as a dev dependency. Happy to revert that bit if you don't like it.

### CODE REVIEW

File: test/Main.purs [hunk 1]
```diff
@@ -4,19 +4,27 @@ import Prelude
...
+import Data.Unfoldable (replicate)
+import Test.Unit (test, suite)
+import Test.Unit.Assert (assert)
+import Test.Unit.Console (TESTOUTPUT)
+import Test.Unit.Main (runTest)
 
 import Control.Monad.Aff.Reattempt (reattempt)
 
+ignore :: forall m a. Monad m => m a -> m Unit
+ignore m = m >>= (\_ -> pure unit)
```
Comment (cprichmond) [inline test/Main.purs]: I tried to ignore this but couldn't... where is it used?

Comment (daniel-chambers) [inline test/Main.purs]: Good catch; that's some trash I was experimenting with and forgot to delete.

**Comment by garyb** [comment]
Thanks!

**Comment by daniel-chambers** [comment]
My pleasure. :)

**Pull request merged**